bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -88,23 +83,7 @@ export default injectIntl(withModalMounter(withTracker(({ intl, baseControls })
     },
   });
 
-  // forcefully log out when the meeting ends
-  Meetings.find({ meetingId: Auth.meetingID }).observeChanges({
-    removed() {
-      if (isMeetingBreakout) {
-        Auth.clearCredentials().then(window.close);
-      } else {
-        endMeeting('410');
-      }
-    },
-  });
-
-  // Close the window when the current breakout room ends
-  Breakouts.find({ breakoutId: Auth.meetingID }).observeChanges({
-    removed() {
-      Auth.clearCredentials().then(window.close);
-    },
-  });
+  if (meeting && meeting.meetingEnded) window.close();
```

Comment:
We are removing `.observeChanges{}`. Are you sure we are handling this reactively?

Reply:
Yes

Reply:
If this code is in a withTracker it will remain active